srml/support/src/traits.rs

```diff
@@ -648,3 +648,24 @@ pub trait ChangeMembers<AccountId> {
 impl<T> ChangeMembers<T> for () {
 	fn change_members(_incoming: &[T], _outgoing: &[T], _new_set: &[T]) {}
 }
+
+/// A generic trait for reporting slashing violations.
+pub trait ReportSlash<Misbehaved, Reporter, Hash> {
+	/// Reports slashing for a misconduct where `to_slash` is the misbehaved entities and
```

Comment:
I'm not native english speaker, but it sounds better to me `to_slash *are* the misbehaved entities`

Reply:
yeah, it is a weird sentence but I think regarded the `to_slash` as a singular (single variable) thus `is` but I can change it to `are`